src/util/misc.ts

```diff
@@ -37,6 +37,23 @@ import { cos } from './index.ts';
       return Math.sin(angle);
     },
 
+    /**
+     * Returns the square root of the sum of squares of its arguments\
+     * Chrome implements `Math#hypot` with a calculation that affects percision so we hard code it as a util
+     * @see https://stackoverflow.com/questions/62931950/different-results-of-math-hypot-on-chrome-and-firefox
+     * @static
+     * @memberOf fabric.util
+     * @param {...number}
+     * @returns {number}
+     */
+    hypot: function () {
+      var sumOfSquares = 0;
+      for (var i = 0; i < arguments.length; i++) {
```

Comment:
es6 now!
I am on it

Reply:
[813358e](https://github.com/fabricjs/fabric.js/pull/8083/commits/813358e8f12de02ffe30b47f7a512543172b3645)